src/udev/udevd.c

```diff
@@ -770,6 +776,8 @@ static int is_device_busy(Manager *manager, struct event *event) {
         return false;
 
 set_delaying_seqnum:
+        device_get_seqnum(event->dev, &seqnum);
```

Comment:
Missing error handling.

Reply:
fixed. I emit SEQNUM=0 in the error case, not entirely sure this is the best option.